src/imagery/i.ann.maskrcnn/maskrcnnlib/model.py

```diff
@@ -29,10 +29,15 @@
 import utils
 
 # Requires TensorFlow 1.3+ and Keras 2.0.8+.
-from distutils.version import LooseVersion
-
-assert LooseVersion(tf.__version__) >= LooseVersion("1.3")
-assert LooseVersion(keras.__version__) >= LooseVersion("2.0.8")
+version_pattern = "([0-9]{1,3}).([0-9]{1,4})"
```

Comment:
```suggestion
version_pattern = "([0-9]{1}).([0-9]{1,2})"
```

Reply:
I agree that the likelyhood that broad version ranges are needed is low. However, it should not hurt either and with wider ranges you are less vulnerable. But I have nothing against reducing the range if you prefer that...

Reply:
Okay. In the end, I have no strong opinion on that. So let's keep the broader ranges. Approving the PR.